components/sections/osmo-cta/backers-section.tsx

```diff
@@ -89,75 +89,67 @@ const backers = [
 ];
 
 export default function BackersSection() {
-  const [isExpanded, setIsExpanded] = useState(false);
-
   return (
-    <>
-      <div className="flex flex-col items-center justify-center gap-4 self-stretch xl:py-3">
-        <span className="leading-6.25 text-alpha-60">Backers include</span>
-        <div
-          className={cn(
-            "relative h-[184px] overflow-hidden transition-all md:h-full",
-            {
-              "h-full": isExpanded,
-            },
-          )}
-        >
-          <div className="flex flex-col gap-2 md:flex-row md:flex-wrap md:content-center md:items-center md:justify-center md:gap-3 md:self-stretch">
-            {backers.map(({ imageUri, name, isPortrait, isTextSm }) => {
-              return (
-                <div
-                  key={name}
-                  className="relative flex h-14 w-[342px] items-center justify-center self-stretch rounded-lg border border-solid border-osmoverse-650 bg-osmoverse-775 sm:w-[416px] md:w-max"
-                >
-                  {isPortrait ? (
-                    <div className="flex h-auto w-full items-center justify-center gap-2.5 md:px-6">
-                      <Image
-                        src={imageUri}
-                        className="h-auto w-auto"
-                        width={42}
-                        height={34}
-                        alt={name}
-                      />
-                      <span
-                        className={cn("text-neutral-100", {
-                          "leading-5.5 text-sm": isTextSm,
-                          "leading-6.25 text-base": !isTextSm,
-                        })}
-                      >
-                        {name}
-                      </span>
-                    </div>
-                  ) : (
+    <div className="flex flex-col items-center justify-center gap-4 self-stretch xl:py-3">
+      <span className="leading-6.25 text-alpha-60">Backers include</span>
+      <div className="flex flex-col-reverse items-center justify-center">
+        <div className="peer">
+          <Switch className="flex items-center p-4 md:hidden">
```

Comment:
Do we really need a switch dep? Can we just use a simple checkbox wrapped liked a button? 

https://tailwindcss.com/docs/hover-focus-and-other-states#checked